src/loaders/genome_attributes/parse_computed_genome_attribs.py

```diff
@@ -0,0 +1,250 @@
+
+usage: parse_computed_genome_attribs.py [-h] --load_ver LOAD_VER --kbase_collection KBASE_COLLECTION
+                                        [--tools TOOLS [TOOLS ...]] [--root_dir ROOT_DIR] [-o OUTPUT]
+
+options:
+  -h, --help            show this help message and exit
+
+required named arguments:
+  --load_ver LOAD_VER   KBase load version (e.g. r207.kbase.1).
+  --kbase_collection KBASE_COLLECTION
+                        KBase collection identifier name.
+
+optional arguments:
+  --tools TOOLS [TOOLS ...]
+                        Extract results from tools. (default: retrieve all available sub-directories in the [load_ver]
+                        directory)
+  --root_dir ROOT_DIR   Root directory for the collections project. (default: /global/cfs/cdirs/kbase/collections)
+  -o OUTPUT, --output OUTPUT
+                        output JSON file path (default: computed_genome_attribs.json)
+
+"""
+import argparse
+import copy
+import os
+import sys
+
+import pandas as pd
+
+from src.loaders.common import loader_common_names
+from src.loaders.gtdb.gtdb_loader_helper import convert_to_json, init_genome_atrri_doc, merge_docs
+
+# Default result file name for parsed computed genome attributes data for arango import
+COMPUTED_GENOME_ATTR_FILE = "computed_genome_attribs.json"
+
+# The following features will be extracted from the CheckM2 result quality_report.tsv file as computed genome attributes
+# If empty, select all available fields
+SELECTED_CHECKM2_FEATURES = {'Completeness', 'Contamination'}
+
+# The following features will be extracted from the GTDB-TK summary file
+# ('gtdbtk.ar53.summary.tsv' or 'gtdbtk.bac120.summary.tsv') as computed genome attributes
+# If empty, select all available fields
+SELECTED_GTDBTK_SUMMARY_FEATURES = {}
+
+
+def _locate_dir(root_dir, kbase_collection, load_ver, check_exists=False, tool=''):
+    result_dir = os.path.join(root_dir, loader_common_names.COLLECTION_DATA_DIR, kbase_collection, load_ver, tool)
+
+    if check_exists and not (os.path.exists(result_dir) and os.path.isdir(result_dir)):
+        raise ValueError(f"Result directory for computed genome attributes of "
+                         f"Kbase Collection: {kbase_collection} and Load Version: {load_ver} could not be found.")
+
+    return result_dir
+
+
+def _read_tsv_as_df(file_path, features, genome_id_col=None):
+    # Retrieve the desired fields from a TSV file and return the data in a dataframe
+
+    selected_cols = copy.deepcopy(features) if features else None
+
+    if selected_cols and genome_id_col:
+        selected_cols.add(genome_id_col)
+
+    df = pd.read_csv(file_path, sep='\t', keep_default_na=False, usecols=selected_cols)
+
+    return df
+
+
+def _create_doc(row, kbase_collection, load_version, genome_id, features, pre_fix='KBase'):
+    # Select specific columns and prepare them for import into Arango
+
+    # NOTE: The selected column names will have a prefix added to them to distinguish them from the original
```

Comment:
is it necessary or making UI looks worse?

Reply:
I'm not sure I understand what's going on here. Why are the columns being prefixed?

Reply:
nv, using an empty string as the default prefix now. I was trying to address the possibility of multiple tools producing results with the same name (e.g. checkm_completeness vs checkm2_completeness). However, it may not be necessary at this time.

Reply:
:+1: